prac_06/guitars.py

```diff
@@ -0,0 +1,39 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    print("My Guitars!")
+    guitars = []
+
+    name = get_input("Name: ")
+    while name:
+        year = get_input("Year: ")
+        cost = get_input("Cost: ")
+        guitars.append(Guitar(name, year, cost))
+        print(f"{name} ({year}) : ${cost} added")
+        name = get_input("Name: ")
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+    print("These are my guitars: ")
+    counter = 1
+    for guitar in guitars:
+        if guitar.is_vintage():
+            vintage_string = "(vintage)"
+        else:
+            vintage_string = ""
+        print(f"Guitar {counter}: {guitar.name:>20} ({guitar.year}), worth ${guitar.cost:10,.2f} {vintage_string}")
+        counter += 1
```

Comment:
If you use enumerate function, there is no need for a counter variable. I.e. 'for i, guitar in enumerate(guitars):'
can combine lines 19-23 into one line using line comprehension.

Reply:
Wasn't able to combine it all into one line, however I did get the enumerate function to work